Fix #8: proper newlines replacement

krizhanovsky opened the pull request:
Fix newlines replacement:
1. Do not put a space between opened and closed tags.
2. Do put spaces between various punctuation characters and words.

krizhanovsky committed:
Fix newlines replacement:
1. Do not put a space between opened and closed tags.
2. Do put spaces between various punctuation characters and words.

leejo commented:
Looks like this test breaks tests - can you fix that? Also, probably makes sense to also add a test for this use case - I suggest the examples you gave in #8. Ta!

krizhanovsky committed:
Fix tests after the previous commit

krizhanovsky committed:
Add dash and comma-ended lines to remove_newlines test; fix for the original fix as it was wrong

krizhanovsky committed:
Fix js newlines test

krizhanovsky commented:
Sorry for the mess in the commits. I added the new test, adjusted existing tests and fixed my original fix which was actually wrong. I have no idea about the last one problem on Travis-CI: https://travis-ci.org/leejo/html-packer-perl/jobs/280625238 - seems like a problem in Cover.pm?.. Could you please have a look?

Thanks!

leejo commented:
Looks like some 5.8 issue, I would suggest removing 5.8 from the `.travis.yml` and we can ignore that one on CI.

krizhanovsky committed:
Remove 5.8 from CI

leejo commented:
Thanks! v2.06 will be on its way to CPAN shortly
